### ISSUE TITLE
Email addresses are automatically converted to hyperlinks, but only if ending in .com

### ISSUE THREAD

**Issue by mastazi** [fabiospampinato/notable#1668]
- **OS Version**: Mac OS 13.2.1
- **Notable Version**: 1.8.4

### Current behavior

email addresses are automatically converted to hyperlinks after exiting edit mode, but **only email addresses that end in .com**

for example

example@domain.com <--- this will become a hyperlink
example@domain.com.au <--- this won't

Here is a screenshot:
<img width="249" alt="Screenshot 2023-03-17 at 3 32 30 pm" src="https://user-images.githubusercontent.com/1661655/225813260-d59c8aee-2a7b-473a-95f8-2c6ae01ba439.png">

### Expected behavior

I would like to just disable this feature and only have hyperlinks occurring when I explicitly create a hyperlink, not automatically when the app wants it to happen

**Comment by fabiospampinato** [comment]
Check out beta10, it should work like that there: https://github.com/notable/notable-insiders/releases/tag/v1.9.0-beta.10

**Comment by mastazi** [comment]
Yes, that's perfect thank you Fabio, closing this and looking forward to 1.9.x release!

**Comment by fabiospampinato** [comment]
Let me keep this open as I want to double-check that emails can be turned into links automatically when setting the app to do so.
